configs/kde/root-image/etc/lightdm/lightdm.conf

```diff
@@ -99,7 +99,7 @@ greeter-session=lightdm-webkit2-greeter
 #greeter-allow-guest=true
 #greeter-show-manual-login=false
 #greeter-show-remote-login=true
-user-session=gnome
+user-session=kde
```

Comment:
@lots0logs Ah it's plasma.desktop - [well what the Sddm page says](https://wiki.archlinux.org/index.php/SDDM#Configuration), I'll fix all the notes up when I get home.